shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.h

```diff
@@ -35,6 +35,8 @@
 
 - (void)await;
 
+- (void)setAllowPauseAfterVsync:(BOOL)allowPause;
```

Comment:
Needs a docstring and should be a property for idiomatic objc.

Reply:
You mean like this ?
```objective-c
@interface VSyncClient

@property(nonatomic, assign)allowPauseAfterVsync; /// here?

@end

@implementation VSyncClient {
  flutter::VsyncWaiter::Callback callback_;
  fml::scoped_nsobject<CADisplayLink> display_link_;
  double current_refresh_rate_;
  BOOL allow_pause_after_vsync_;
}
```

Reply:
Yea, but you don't need the variable in the .m file.  `allow_pause_after_vsync_` is generated for you automatically.

Reply:
Emm I got the error
```
../../flutter/shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm:66:5: error: use of undeclared identifier 'allow_pause_after_vsync_'; did you mean '_allowPauseAfterVsync'?
    allow_pause_after_vsync_ = true;
    ^~~~~~~~~~~~~~~~~~~~~~~~
    _allowPauseAfterVsync
../../flutter/shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm:50:35: note: '_allowPauseAfterVsync' declared here
@property(nonatomic, assign) BOOL allowPauseAfterVsync;
                                  ^
../../flutter/shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm:107:3: error: use of undeclared identifier 'allow_pause_after_vsync_'; did you mean '_allowPauseAfterVsync'?
```
Seems that I should change `allow_pause_after_vsync_` to `_allowPauseAfterVsync`

Reply:
Done.

Reply:
Sorry, yea I didn't explain it well.  The generated ivar has the name `_$propertyName`.  This is still not a property.  Thanks for adding a docstring.  I wish it would tell me more than what I already can gather from the method name.  Is there somewhere else that explains pausing in the header you can link to?

Reply:
I have change a doc with more details^_^.

Reply:
Done.

Reply:
This still isn't a property =)

Reply:
Now should be ok...😄